### ISSUE TITLE
Errors I get after install.

### ISSUE THREAD

**Issue by ikester7579** [wp-healthcheck/wp-healthcheck#15]
```
Warning: Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/notices/web.php on line 9

Warning: Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/admin/sysinfo.php on line 8

Warning: Illegal string offset 'version' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/admin/sysinfo.php on line 11
```

That is what appears when I go into the plugin panel. Now here's what appears in the log.

```
[08-Feb-2018 09:09:22 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/includes/class-wp-healthcheck.php on line 254
[08-Feb-2018 09:09:22 UTC] PHP Warning:  Illegal string offset 'version' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/includes/class-wp-healthcheck.php on line 259
[08-Feb-2018 09:09:22 UTC] PHP Warning:  Cannot assign an empty string to a string offset in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/includes/class-wp-healthcheck.php on line 259
[08-Feb-2018 09:09:22 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/notices/web.php on line 9
[08-Feb-2018 09:09:22 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/admin/sysinfo.php on line 8
[08-Feb-2018 09:09:22 UTC] PHP Warning:  Illegal string offset 'version' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/admin/sysinfo.php on line 11
[08-Feb-2018 09:09:22 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/admin/sysinfo.php on line 38
[08-Feb-2018 09:12:20 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/includes/class-wp-healthcheck.php on line 254
[08-Feb-2018 09:12:20 UTC] PHP Warning:  Illegal string offset 'version' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/includes/class-wp-healthcheck.php on line 259
[08-Feb-2018 09:12:20 UTC] PHP Warning:  Cannot assign an empty string to a string offset in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/includes/class-wp-healthcheck.php on line 259
[08-Feb-2018 09:12:20 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/notices/web.php on line 9
[08-Feb-2018 09:12:20 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/admin/sysinfo.php on line 8
[08-Feb-2018 09:12:20 UTC] PHP Warning:  Illegal string offset 'version' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/admin/sysinfo.php on line 11
[08-Feb-2018 09:12:20 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/admin/sysinfo.php on line 38
[08-Feb-2018 09:16:43 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/notices/web.php on line 9
[08-Feb-2018 09:16:49 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/notices/web.php on line 9
[08-Feb-2018 09:16:49 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/admin/sysinfo.php on line 8
[08-Feb-2018 09:16:49 UTC] PHP Warning:  Illegal string offset 'version' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/admin/sysinfo.php on line 11
[08-Feb-2018 09:16:49 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/admin/sysinfo.php on line 38
[08-Feb-2018 09:17:31 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/notices/web.php on line 9
[08-Feb-2018 09:17:38 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/notices/web.php on line 9
[08-Feb-2018 09:17:38 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/admin/sysinfo.php on line 8
[08-Feb-2018 09:17:38 UTC] PHP Warning:  Illegal string offset 'version' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/admin/sysinfo.php on line 11
[08-Feb-2018 09:17:38 UTC] PHP Warning:  Illegal string offset 'service' in /home/yech2/public_html/wp-content/plugins/wp-healthcheck/views/admin/sysinfo.php on line 38
```

**Comment by tiagohillebrandt** [comment]
Thanks for the bug report @ikester7579. We'll look into it and get it fixed in our next release.

**Comment by tiagohillebrandt** [comment]
The warnings should have been fixed on 1.2.1 release. Thanks again @ikester7579 :-)
